herbert_core/classes/data_loaders/@loader_nxspe/loader_nxspe.m

```diff
@@ -19,7 +19,8 @@
         % the structure, containing the folder structure of the nxspe file
         % as defined in hdf5 file
         nexus_dataset_info_ = [];
-        %
+        % a structure for instrument information
+        nexus_instrument_ = [];
```

Comment:
See overall review comment  - this line comment is a placeholder

Reply:
I don't understand your comment "this line comment is a placeholder"?

This change does not cover detectors or samples. Just the "primary spectrometer" (all instrument components before the sample).

Reply:
The words "this comment is a placeholder" means that I want to draw attention to something for the whole PR, but it is easy to miss the whole-review comment and so I put a comment here to send you to the whole-PR comment. It is not a comment on line 23. 

Your comment that this change covers only the primary spectrometer is good but it could usefully go in the PR description as well.